kong/plugins/apifortress/schema.lua

```diff
@@ -0,0 +1,15 @@
+-- The plugin schema
+
+local function validateKeys(value)
+	return string.len(value) > 0
+end
+
+return {
+  fields = {
+   	apikey = { required = true, type = "string", func = validateKeys },
+		secret = { required = true, type = "string", func = validateKeys },
+		endpoint = {required = true, type = "string", func = validateKeys},
+		projectId = {required = true , type = "number", func = validateKeys},
```

Comment:
I don't think `validateKeys` is necessary here. As opposed to the strings above, this field is declared a number and is required, thus, the check can be safely avoided, as it will always be true. Unless this should be a string?